tests/ui/loop-match/unwind.rs

```diff
@@ -0,0 +1,49 @@
+// Test that #[const_continue] correctly emits cleanup paths for drops.
```

Comment:
```suggestion
// Test that `#[const_continue]` correctly emits cleanup paths for drops.
//
// Here, we first drop `DropBomb`, causing an unwind. Then `ExitOnDrop` should
// be dropped, causing us to exit with `0` rather than with some non-zero value
// due to the panic, which is what causes the test to pass.
```